lib/src/test/java/com/auth0/jwt/JWTCreatorTest.java

```diff
@@ -157,7 +158,42 @@ public void shouldNotOverwriteKeyIdIfAddedFromRSAAlgorithms() throws Exception {
         String headerJson = new String(Base64.decodeBase64(parts[0]), StandardCharsets.UTF_8);
         assertThat(headerJson, JsonMatcher.hasEntry("kid", "my-key-id"));
     }
-
+    
+    @Test
+    public void shouldAddKeyIdIfAvailableFromECDSAKAlgorithms() throws Exception {
+        ECPrivateKey privateKey = (ECPrivateKey) PemUtils.readPrivateKeyFromFile(PRIVATE_KEY_FILE_EC_256K
+                , "EC");
+        ECDSAKeyProvider provider = mock(ECDSAKeyProvider.class);
+        when(provider.getPrivateKeyId()).thenReturn("my-key-id");
+        when(provider.getPrivateKey()).thenReturn(privateKey);
+        
+        String signed = JWTCreator.init()
+                .sign(Algorithm.ECDSA256K(provider));
+        
+        System.out.println("TOKEN: "+signed);
```

Comment:
```suggestion
        System.out.println("TOKEN: "+signed); 
```
LEFT BY MISTAKE, should be removed.